src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -209,9 +209,10 @@ export class AddressForm extends React.Component<
               <Input
                 id="AddressForm_phoneNumber"
                 title="Phone number"
+                type="tel"
                 description={this.phoneNumberInputDescription()}
-                placeholder="Add phone"
-                pattern="[0-9]*"
+                placeholder="+1-222-222-2222"
+                pattern="[^a-z]+"
```

Comment:
This looks like a good iteration!

Agreed that we should ask about potential validations at the API level. I also wonder if we can add some more structure around country codes. How do other sites support international phone numbers? Can we use a react component that nicely masks the input while sending up clean, structured values?

Reply:
>  How do other sites support international phone numbers? Can we use a react component that nicely masks the input while sending up clean, structured values?

While doing some research, I did find some elegant looking ways to structure international phone number input, but want to focus on the acute problem here for now